# The time point at which this event occurred.
uint64 timestamp

# The starting mode from which this event transitioned.
Mode start_mode

# The end mode of this transition event.
Mode goal_mode
